client/css/style.css

```diff
@@ -1961,6 +1961,14 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	}
 }
 
+@media (min-width: 1610px) {
+	#windows .header .topic,
+	.messages .msg,
+	.sidebar {
+		font-size: 14px;
+	}
+}
+
```

Comment:
@xPaw, @YaManicKill, what do you think about this? Aside of that I'm 👍 but I just wanted to double check if you were okay with this or not.

Reply:
I meant adding a new responsive breakpoint to our CSS. I guess we're fine with that.